File: java/src/main/java/com/dsi/laboreos/model/TipoSuelo.java
package com.dsi.laboreos.model;

public class TipoSuelo {
    private String nombre;
    private String descripcion;
    private Integer numero;

    public TipoSuelo(String nombre, String descripcion, Integer numero) {
        this.nombre = nombre;
        this.descripcion = descripcion;
        this.numero = numero;
    }

    public String getNombre() {
        return nombre;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public Integer getNumero() {
        return numero;
    }
}

File: python/laboreos/models/TipoSuelo.py
from django.db import models


class TipoSuelo(models.Model):
    numero = models.IntegerField(primary_key=True)
    nombre = models.CharField(max_length=100)
    descripcion = models.TextField()
    
    class Meta:
        db_table = 'tipos_suelo'
        verbose_name_plural = 'Tipos de Suelo'
    
    def __str__(self):
        return f"{self.nombre} (Tipo {self.numero})"

